rsmtool/preprocessor.py

```diff
@@ -164,10 +156,9 @@ def generate_default_specs(cls, feature_names):
         Note
         ----
         Since these are default specifications, the values for the
-        `transform` column for each feature will be `"raw"` and the value
-        for the `sign` column will be `1`.
+        "transform" column for each feature will be "raw" and the
+        value for the "sign" column will be 1.
```

Comment:
Shouldn't this be a literal?

```suggestion
        value for the "sign" column will be ``1``.
```